library/general/test/autoinst_issues/issues_presenter_test.rb

```diff
@@ -24,7 +24,7 @@
 require "installation/autoinst_issues/list"
 require "installation/autoinst_issues/issues_presenter"
 
-module Installation
+module Y2Autoinstallation
```

Comment:
I would not use a module name that already exists (to avoid potential problems). You do not even need to use a module at all. Just declaring a DummyMissingSection class should be enough.